### PR TITLE
Dev/update miscellaneous

### PR DESCRIPTION
**Pull request by loutry**

### CODE REVIEW

**Review by djavan-bertrand — APPROVED** [review]

File: data/database/schedule.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "2022-04-25": {
-    "date": "2022-04-25",
```
Comment (benju69) [inline data/database/schedule.json:3]: Vous pouvez remettre la date ?

File: data/database/schedule.json [hunk 2]
```diff
@@ -327,7 +330,6 @@
     ]
   },
   "2022-04-26": {
-    "date": "2022-04-26",
```
[the remark this answers is not in the document]
Comment (djavan-bertrand) [reply data/database/schedule.json]: Done

File: data/database/schedule.json (hunk 1 above)
Comment (djavan-bertrand) [reply data/database/schedule.json:3]: Done

File: data/database/schedule.json [hunk 3]
```diff
@@ -506,9 +506,7 @@
             ]
           },
           {
-            "items": [
-              ""
```
Comment (benju69) [inline data/database/schedule.json]: AH y'a moyen ça aide

**Comment by benju69** [comment]
we can merge i guess

**Comment by benju69** [comment]
So empty strings seems to break everything 😆

**Review by benju69 — APPROVED** [review]

**Pull request merged**